ui/raidboss/data/05-shb/raid/e5s.js

```diff
@@ -140,11 +140,11 @@
     },
     {
       id: 'E5S Crippling Blow',
-      regex: Regexes.ability({ id: '4BCA', source: 'Ramuh' }),
-      regexDe: Regexes.ability({ id: '4BCA', source: 'Ramuh' }),
-      regexFr: Regexes.ability({ id: '4BCA', source: 'Ramuh' }),
-      regexJa: Regexes.ability({ id: '4BCA', source: 'ラムウ' }),
-      regexKo: Regexes.ability({ id: '4BCA', source: '라무' }),
+      regex: Regexes.startsUsing({ id: '4BCA', source: 'Ramuh', capture: true }),
```

Comment:
can you remove capture: true? it's implicit and gets skipped everywhere else.

Reply:
fixed